### DISCUSSION TITLE
Pods Get Killed with SIGSEGV

### DISCUSSION THREAD

**Discussion by 1995parham** [emqx/emqx#6788] [Q&A] [answered]
We are using version 4.3.11 of emqx on kubernetes. pods randomly killed with exit code 139 (Segmentation Fault). I am forcing all nodes to use the latest hash available on docker-hub. Do you have any idea what is happening? cloud it be related to erlang beam configuration?

**Comment by qzhuyan** [comment]
Could you provide the docker image sha256 hash you use?

are you running on x86 platform?

Could you provide pod logs?

**Reply by 1995parham** [reply]
Yes, we are on x86.

```
sha256:f618dfc834fab14911bc37e24dc834cbffb010b8af9e5c6861721ad0429f2a20
```

About the logs, I will share it.

**Reply by 1995parham** [reply]
I have checked the logs of our recent restarted pod but it doesn't have any log just auth failure related to the unauthorized clients. by the way, based on `emqx_vm_cpu_use` this pod has 62% usage.

**Comment by 1995parham** [comment, marked as answer]
it seems fixed after recreating the whole stateful set.
